stores/migrations/sqlite/main/migration_00002_prune_slabs_trigger.sql

```diff
@@ -0,0 +1,71 @@
+-- prune manually before creating trigger
+DELETE FROM slabs
+WHERE id IN (
+  SELECT slabs.id
+  FROM slabs
+  LEFT JOIN slices ON slices.db_slab_id = slabs.id
+  WHERE slices.db_object_id IS NULL
+  AND slices.db_multipart_part_id IS NULL
+  AND slabs.db_buffered_slab_id IS NULL
+);
+
+PRAGMA foreign_keys=off;
+-- remove ON DELETE CASCADE from slices
+CREATE TABLE `slices_temp` (`id` integer PRIMARY KEY AUTOINCREMENT,`created_at` datetime,`db_object_id` integer,`object_index` integer,`db_multipart_part_id` integer,`db_slab_id` integer,`offset` integer,`length` integer,CONSTRAINT `fk_objects_slabs` FOREIGN KEY (`db_object_id`) REFERENCES `objects`(`id`),CONSTRAINT `fk_multipart_parts_slabs` FOREIGN KEY (`db_multipart_part_id`) REFERENCES `multipart_parts`(`id`),CONSTRAINT `fk_slabs_slices` FOREIGN KEY (`db_slab_id`) REFERENCES `slabs`(`id`));
+INSERT INTO slices_temp SELECT * FROM slices;
+DROP TABLE slices;
+ALTER TABLE slices_temp RENAME TO slices;
+
+CREATE INDEX `idx_slices_object_index` ON `slices`(`object_index`);
+CREATE INDEX `idx_slices_db_object_id` ON `slices`(`db_object_id`);
+CREATE INDEX `idx_slices_db_slab_id` ON `slices`(`db_slab_id`);
+CREATE INDEX `idx_slices_db_multipart_part_id` ON `slices`(`db_multipart_part_id`);
+
+-- remove ON DELETE CASCADE from multipart_parts
```

Comment:
I didn't see this change in the mysql schema.sql I think, we only alter `CREATE TABLE `slices` (`

Reply:
Yeah we alter it to not create the constraint with `ON DELETE CASCADE`. 
The schema is only applied on an empty database so we don't need to explicitly remove anything.

Reply:
I think you misunderstood, on line 313 in the mysql schema there is `CONSTRAINT `fk_multipart_uploads_parts` FOREIGN KEY (`db_multipart_upload_id`) REFERENCES `multipart_uploads` (`id`) ON DELETE CASCADE`. My comment should've said "I didn't see this change _reflected_ in the MySQL schema...